perfdash/perfdash.go

```diff
@@ -161,29 +37,14 @@ func main() {
 	fmt.Print("Starting perfdash...\n")
 	flag.Parse()
 
-	var downloader Downloader
-	downloader = NewGoogleGCSDownloader(*startFrom)
-
-	if !*www {
-		buildLatency, resources, methods, err := downloader.getData()
-		if err != nil {
-			fmt.Printf("Failed to get data: %v\n", err)
-			os.Exit(1)
-		}
-		for _, v := range buildLatency {
-			generateCSV(v, resources, methods, os.Stdout)
-		}
-		return
-	}
+	downloader := NewGoogleGCSDownloader(*builds)
```

Comment:
Validate the number of builds. There's probably no reason to allow more than a 100 (and memory usage grows with this number)